apps/sms/views/shared/style/panel.css

```diff
@@ -9,56 +9,17 @@
 
   background: white;
 
-  animation: .2s both linear;
+  animation: var(--transition-duration) both ease;
 }
-html[dir="ltr"] .panel { transform: translate(100%); }
-html[dir="rtl"] .panel { transform: translate(-100%); }
 
-/**
- * adding html[dir] to make the specificity of this rule high enough to
- * overwrite the default translation for element with class panel just above,
- * without needing to put a !important
- */
-html[dir] .panel-active {
```

Comment:
actually I don't think this was necessary; what's necessary is having the right animations going on depending on the LTR/RTL status. But changing where the panel is hidden is IMO useless. What do you think @autra ?